src/Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights/Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights.csproj

```diff
@@ -27,7 +27,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.1.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.2.0" />
```

Comment:
Added https://github.com/Azure/diagnostics-eventflow/issues/71 to track this

I do want people from outside Microsoft to be able to build and contribute to EventFlow, so I do not want to take dependencies on non-public stuff